dashboard/README.md

```diff
@@ -0,0 +1,52 @@
+# Milpa Dashboard
+The Milpa Dashboard is a web UI that allows users to view the cloud costs associated with a Milpa cluster and also view the resources running on the cluster.
+
```

Comment:
It might be useful to insert a screenshot of dashboard landing page here so that users have a visual of what dashboard offers.